plotly/utils.py

```diff
@@ -32,6 +33,30 @@
 lock = threading.Lock()
 
 
+http_msg = (
+    "The plotly_domain and plotly_api_domain of your config file must start "
+    "with 'https', 'http'.\n If you are not using On-Prem then run the "
```

Comment:
Why include `http` here? --->  "with 'https', 'http'. 

Also I would suggest using `On-Premise` instead of `On-Prem` to be consistent with the rest of the codebase.

Reply:
That was a typo. I meant to type `'http', not 'https'`